CHANGELOG.md

```diff
@@ -47,6 +47,8 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
   as that package has not yet been updated in the upstream `nginx` Docker image.
   (#88)
 
+- Changed docker-build scripts for easier windows building. (#95)
```

Comment:
nit: Could use the filename "Dockerfile" instead of "docker-build scripts" to be more precise. IMO I understand "Dockerfile" better

```suggestion
- Changed Dockerfile for easier windows building. (#95)
```